doc: approtect: minor style edits (#10292)
nrfconnect/sdk-nrf: doc/nrf/app_dev/ap_protect/index.rst

.. list-table:: AP-Protect implementations
   :header-rows: 1

   * - AP-Protect implementation type
     - Default factory state
     - How to enable
     - How to disable
   * - Hardware
     - Disabled

>      - Writing ``UICR.APPROTECT`` to ``Enabled`` and performing reset.

Reviewer: ```suggestion
     - Writing ``UICR.APPROTECT`` to ``Enabled`` and performing a reset.
```

     - Issuing an ``ERASEALL`` command via CTRL-AP.
       This command erases the flash, UICR, and RAM, including ``UICR.APPROTECT``.
   * - Hardware and software
     - Enabled

>      - After it is disabled, a reset or a wake enables the access port protection again.

Reviewer: ```suggestion
     - When disabled, a reset or a wake enables the access port protection again.
```
This sentence feels like it was taken from a more running text that told how to disable it.

Should consideration be given to the second half of the disabling instructions?

>      - Issuing an ``ERASEALL`` command via CTRL-AP.

Reviewer: Doesn't this erase `UICR.APPROTECT` being programmed to `HwDisabled`, and thus it never stays disabled?

Or am I not understandinf the table correctly?

Author: You erase, then write `UICR.APPROTECT` to `HwDisabled`, program, and avoid using a reset or a wake until you're done working and want to re-enable AP-Protect.
